pytest_tests/tests/services/s3_gate/test_s3_policy.py

```diff
@@ -86,6 +86,66 @@ def test_s3_bucket_location(self, simple_object_size):
             )
             assert copies_2 == 3
 
+    @allure.title("Test S3: Verify bucket creation with policies from config file")
+    def test_s3_bucket_location_from_config_file(self, simple_object_size):
+        if self.neofs_env.s3_gw._get_version() <= "0.31.1":
+            pytest.skip("This test runs only on post 0.31.1 S3 gw version")
+        file_path_1 = generate_file(simple_object_size)
+        file_name_1 = object_key_from_file_path(file_path_1)
+        file_path_2 = generate_file(simple_object_size)
+        file_name_2 = object_key_from_file_path(file_path_2)
+
+        with allure.step("Create two buckets with different bucket configuration"):
+            bucket_1 = s3_gate_bucket.create_bucket_s3(self.s3_client, bucket_configuration="select")
+            set_bucket_versioning(self.s3_client, bucket_1, s3_gate_bucket.VersioningStatus.ENABLED)
+            bucket_2 = s3_gate_bucket.create_bucket_s3(self.s3_client, bucket_configuration="rep-2")
+            set_bucket_versioning(self.s3_client, bucket_2, s3_gate_bucket.VersioningStatus.ENABLED)
+            list_buckets = s3_gate_bucket.list_buckets_s3(self.s3_client)
+            assert (
+                bucket_1 in list_buckets and bucket_2 in list_buckets
+            ), f"Expected two buckets {bucket_1, bucket_2}, got {list_buckets}"
+
+            head_1 = s3_gate_bucket.head_bucket(self.s3_client, bucket_1)
+            head_2 = s3_gate_bucket.head_bucket(self.s3_client, bucket_2)
+            assert head_1 == {} or head_1.get("HEAD") is None, "Expected head is empty"
+            assert head_2 == {} or head_2.get("HEAD") is None, "Expected head is empty"
+
+        with allure.step("Put objects into buckets"):
+            version_id_1 = s3_gate_object.put_object_s3(self.s3_client, bucket_1, file_path_1)
+            version_id_2 = s3_gate_object.put_object_s3(self.s3_client, bucket_2, file_path_2)
+            check_objects_in_bucket(self.s3_client, bucket_1, [file_name_1])
+            check_objects_in_bucket(self.s3_client, bucket_2, [file_name_2])
+
+        with allure.step("Check bucket location"):
+            bucket_loc_1 = s3_gate_bucket.get_bucket_location(self.s3_client, bucket_1)
+            bucket_loc_2 = s3_gate_bucket.get_bucket_location(self.s3_client, bucket_2)
+            assert bucket_loc_1 == "select"
+            assert bucket_loc_2 == "rep-2"
+
+        with allure.step("Check object policy"):
```

Comment:
Can you also check _container_ policy? If you're to get the container, it'll be there.